doc/wg/network/README.md

```diff
@@ -0,0 +1,43 @@
+Network Working Group (NWG)
+=================================
+
+- Working Group Charter
+- Adopted 7/28/2023
+
+## Goals
+
+The goals of the Network Tock Working Group (NWG) are to:
+
+- design a set of interfaces (traits) for networking
+- design the user land interfaces for networking
+- define how buffer management should be implemented
+
+## Members
+
+- Alexandru Radovici, OxidOS Automotive
+- Brandon Ghena, Northwestern University
+- Leon Schuermann, Princeton University
+- Tyler Potyondy, UCSD
+- Ioan-Cristian Cirstea, OxidOS Automotive
+
+## Membership and Communication
+
+The networking working group membership is a subset of the people who have
+commit (pull request merge) permissions on the Tock repository. It is
+intended to be a smaller group that represents the major perspectives
+and issues, rather than a complete group. Group membership is decided by
+the group: the exact process is not yet determined and may organically
+evolve as the group gains momentum.
+
+The group has a weekly teleconference call. All working group members
```

Comment:
```suggestion
The group has a biweekly teleconference call. All working group members
```

Assuming we stick to what we talked about at TockWorld 6. Not strictly opposed to weekly though.

Reply:
"biweekly" is actually ambiguous in English and means either twice a week or once every two weeks. Both meanings are common which really just means that the word is meaningless.

```suggestion
The group has a teleconference call once every two weeks. All working group members
```